static/downloadables/runPrsCLI.sh

```diff
@@ -73,6 +72,13 @@ version="1.7.0"
 #   Exclude duplicated snps and sex-based snps unless sex-based snps are requested (-g parameter)
 #
 #
+#   11/10/2021
+#
+#   adding in handling for pvalue annotationa and beta values
```

Comment:
change "annotationa" to "annotation"

Reply:
change logs are also normally in past tense ("added", "switched" ("changed" is probably a better word), and "added" again)